metadata-ingestion/tests/integration/powerbi/golden_test_admin_only.json

```diff
@@ -1,12 +1,46 @@
 [
+{
```

Comment:
Could you please check why user related MCP got added, ideally these should not get ingested.

custom-properties related changes in golden file is expected.